python/core/qgsdistancearea.sip

```diff
@@ -146,6 +146,21 @@ class QgsDistanceArea
      */
     double measureLine( const QgsPoint& p1, const QgsPoint& p2 ) const;
 
+    /**
+     * calculates distance from one point with distance in meters and azimuth (direction)
+     * based on inverse Vincenty's formula when Geographic
+     * otherwise QgsPoint.project() will be called after QgsUnitTypes::fromUnitToUnitFactor() has been applied to the distance
+     * note: usage:
+     *  QgsDistanceArea myDa;
+      * myDa.setSourceCrs( layer->crs() );
+     * \since QGIS 3.0
+     * @param p1 start point [can be Cartesian or Geographic]
+     * @param distance expected to be in meters
+     * @param projected_point calculated projected point
+     * @return distance in mapUnits
+     */
+    double measureLineProjected( const QgsPoint &p1, double distance = 1, double azimuth = M_PI / 2, QgsPoint *projected_point = nullptr );
```

Comment:
Check the pyqt docs - http://pyqt.sourceforge.net/Docs/sip4/annotations.html

"This boolean annotation is used to specify that the corresponding argument (which should be a pointer type) is used by the function to return a value as an element of a tuple."

Without it projectedPoint won't be usable from Python